source/ui.py

```diff
@@ -103,37 +158,59 @@ def browseableMessage(message: str, title: Optional[str] = None, isHtml: bool =
 
 	htmlFileName = os.path.join(globalVars.appDir, "message.html")
 	if not os.path.isfile(htmlFileName):
+		_warnBrowsableMessageComponentFailure(title)
 		raise LookupError(htmlFileName)
+
 	moniker = POINTER(IUnknown)()
-	windll.urlmon.CreateURLMonikerEx(0, htmlFileName, byref(moniker), URL_MK_UNIFORM)
-	if not title:
-		# Translators: The title for the dialog used to present general NVDA messages in browse mode.
-		title = _("NVDA Message")
+	try:
+		windll.urlmon.CreateURLMonikerEx(0, htmlFileName, byref(moniker), URL_MK_UNIFORM)
+	except Exception as e:
+		log.error(f"Failed to create URL moniker: {e}")
+		_warnBrowsableMessageComponentFailure(title)
+		return
+
 	if not isHtml:
 		message = f"<pre>{escape(message)}</pre>"
+
 	try:
 		d = comtypes.client.CreateObject("Scripting.Dictionary")
 	except (COMError, OSError):
 		log.error("Scripting.Dictionary component unavailable", exc_info=True)
-		# Store the module level message function in a new variable since it is masked by a local variable with
-		# the same name
-		messageFunction = globals()["message"]
-		# Translators: reported when unable to display a browsable message.
-		messageFunction(_("Unable to display browseable message"))
+		_warnBrowsableMessageComponentFailure(title)
 		return
 	d.add("title", title)
 	d.add("message", message)
+	# Translators: A notice to the user that a copy operation succeeded.
+	d.add("copySuccessfulAlertText", _("Text copied."))
+	# Translators: A notice to the user that a copy operation failed.
+	d.add("copyFailedAlertText", _("Couldn't copy to clipboard."))
+	if closeButton:
+		# Translators: The text of a button which closes the window.
+		d.add("closeButtonText", _("Close"))
+	if copyButton:
+		# Translators: The label of a button to copy the text of the window to the clipboard.
+		d.add("copyButtonText", _("Copy"))
+		# Translators: A portion of an accessibility label for the "Copy" button,
+		# describing the key to press to activate the button. Currently, this key may only be Alt+C.
```

Comment:
If this is only the case, should it be translatable?
what can translators change it to?

Reply:
Is the key known as "Alt" the same in all languages? Can all keyboards generate an `Alt+C`, or will some translations need to set it to the empty string because that key can not be used there?

These were my thoughts. I am only familiar with English language and keyboards.
If they are invalid, then I will remove the translatability of that string.
It was the best I could come up with short of making the key remappable, which is impractical as far as I can tell.

Reply:
it's probably worth noting that it should be set to an empty string if alt+c is not supported in the local keyboard